dashboard/client/pages/login.tsx

```diff
@@ -1,14 +1,44 @@
 import * as React from 'react'
 
-import GoogleButton from '../components/GoogleButton'
+import Router from 'next/router'
+import { auth } from '../lib/gql/query'
+
 import Layout from '../components/Layout'
 
-const Login: React.SFC = () => (
-  <Layout>
-    <main className="main" id="main__login">
-      <GoogleButton />
-    </main>
-  </Layout>
-)
+const Login = ({ query }) => {
+  const [user, setUser] = React.useState()
+
+  React.useEffect(() => {
+    const isBrowser = typeof window !== undefined
+
+    if (isBrowser) {
+      ;(async () => {
```

Comment:
we need to clean up after the `await` call in case this component unmounts

Reply:
dude this sent me down a rabbit hole bc i was convinced you only have to do that for subscriptions, good to know 👌

Reply:
<img src="https://media.giphy.com/media/3ohzdLQUbKEu47o9Ww/giphy.gif" width="50%" height="50%"  />

lmfao